java/src/main/java/com/logicalclocks/hsfs/metadata/FeatureViewApi.java

```diff
@@ -75,7 +75,19 @@ public FeatureView get(FeatureStore featureStore, String name, Integer version)
 
     LOGGER.info("Sending metadata request: " + uri);
     HopsworksClient hopsworksClient = HopsworksClient.getInstance();
-    return hopsworksClient.handleRequest(request, FeatureView.class);
+    try {
+      return hopsworksClient.handleRequest(request, FeatureView.class);
+    } catch (IOException e) {
+      if (e.getMessage().contains("\"errorCode\":270009")) {
```

Comment:
Everything looks good, but isn't the same error code given even when trying to receive FV that doesn't exist? (not only when FG is deleted)

Reply:
```
HTTP code: 404, HTTP reason: Not Found, body: b'{"type":"restApiJsonResponse","errorCode":270181,"errorMsg":"Feature view wasn\'t found.","usrMsg":"There exists no feature view with the name test_delete and version 10."}', error code: 270181, error msg: Feature view wasn't found., user msg: There exists no feature view with the name test_delete and version 10.
```
You get this when FV does not exist.

Reply:
yeah you are right, `270009` is for FG not found `HTTP code: 404, HTTP reason: Not Found, body: b'{"type":"restApiJsonResponse","errorCode":270009,"errorMsg":"Featuregroup wasn\'t found.","usrMsg":"feature group name: fg_0_ffffsc feature group version: 1"}', error code: 270009, error msg: Featuregroup wasn't found., user msg: feature group name: fg_0_ffffsc feature group version: 1`